packages/react/collapsible/src/Collapsible.tsx

```diff
@@ -32,7 +32,7 @@ interface CollapsibleProps extends PrimitiveDivProps {
   defaultOpen?: boolean;
   open?: boolean;
   disabled?: boolean;
-  onOpenChange?(open?: boolean): void;
+  onOpenChange?(open: boolean): void;
```

Comment:
While fixing this I realised this would error:

```jsx
const [open, setOpen] = React.useState(false);

<Collapsible ... onOpenChange={setOpen} />
```

It errors because the `onOpenChange` expects `setOpen` to accept `undefined` but as far as I know, this would/should always give them a boolean.